csharp/Platform.Numbers/Math[T].cs

```diff
@@ -1,4 +1,4 @@
-﻿using System;
```

Comment:
UTF-8 BOM symbol should not be removed. Try to update your code editor's settings.